### PR TITLE
Only allow Pint 0.21

### PR DESCRIPTION
**Pull request by mattwthompson**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

**Review by j-wags — APPROVED** [review]

**Review by j-wags — COMMENTED** [review]
Oh, wait, bump the build number too

**Comment by mattwthompson** [comment]
Oops, good catch

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ requirements:
   run:
     - python >=3.8
     - numpy
-    - pint >=0.20.1|=0.21
```
Comment (mattwthompson) [inline recipe/meta.yaml:24]: In hindsight this was a poor choice; there was nothing between 0.20.1 and 0.21 so the second option did nothing

**Pull request merged**